packages/cyberstorm/src/components/Layout/Settings/Connections/Connections.module.css

```diff
@@ -42,18 +42,18 @@
   background-color: var(--color-surface--1);
 }
 
-.connectionTypeInfo > svg {
-  min-width: 40px;
-  min-height: 40px;
-}
-
 .connectionTypeInfo {
   display: flex;
   gap: var(--space--16);
   align-items: center;
   justify-content: center;
 }
 
+.connectionTypeInfoIcon {
+  width: 3rem;
+  height: 2.5rem;
+}
```

Comment:
I recommend setting the icon constraints as a square with 2.5rem to align everything nicely.